src/npc.cpp

```diff
@@ -34,6 +34,7 @@ npc::npc()
  wandx = 0;
  wandy = 0;
  wandf = 0;
+ goal = tripoint( 0, 0, 0 );
```

Comment:
Do you need this? The point should have already been constructed with the default constructor, which does already set all members to 0. Same with overmap_special_terrain below.